examples/filter.yml.example

```diff
@@ -1,6 +1,7 @@
 blacklist:
   favicon: ''
 whitelist:
+  country: Germany
```

Comment:
on radiobrowser, I can't find a query to use it. What's your intention?

Reply:
I did this for the unit test, to get only one result from the radiobrowser.get_country_directories(). Although I think i maybe better to setup everything in code rather than relying on the example filter.